### PR TITLE
Change package manager from npm to yarn

### PR DESCRIPTION
**Pull request by Olaleye-Blessing**
### Fixes Issue
Closes #101 

### Changes proposed

- Switch the package manager from npm to yarn

### CODE REVIEW

**Comment by rohitdasu** [comment]
@Olaleye-Blessing could you please update the readme according to `yarn`

**Comment by Olaleye-Blessing** [comment]
@rohitdasu Done

**Comment by rohitdasu** [comment]
@Olaleye-Blessing is there any alternative to npx in yarn universe? just curious

**Comment by Olaleye-Blessing** [comment]
@rohitdasu I don't think so according to [StackOverflow answer](https://stackoverflow.com/a/66612901/11431272).

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ As a community, we always encourage people to share their thoughts and ideas. Do
    ```
 8. Start the application development server
    ```sh
-   npm run dev
+   yarn dev
```
Comment (rupali-codes) [inline README.md:67]:
```suggestion
   yarn run dev
```

Comment (rupali-codes) [inline README.md:67]:
```suggestion
   yarn run dev
```

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Please make the suggested changes

**Comment by Olaleye-Blessing** [comment]
> Please make the suggested changes

The changes have been made.

**Review by rupali-codes — APPROVED** [review]

**Comment by JakubChorzepa** [comment]
> @Olaleye-Blessing is there any alternative to npx in yarn universe? just curious

Hey since we have installed prisma locally we could replace ```npx prisma``` commands with ```yarn prisma```. What do you think about it?

**Comment by Olaleye-Blessing** [comment]
@JakubChorzepa Is it okay to just replace all `npx` with `yarn`?

**Review by rupali-codes — APPROVED** [review]
LGTM

**Review by JakubChorzepa — APPROVED** [review]

**Pull request merged**